scenarios/centos-9/multinode-ci.yml

```diff
@@ -22,6 +22,10 @@ cifmw_update_containers: true
 
 # Enable tempest
 cifmw_run_tests: true
+cifmw_test_operator_stages:
+  - name: tempest
+    type: tempest
+    test_file: /home/zuul/ci-framework-data/artifacts/parameters/zuul-params.yml
```

Comment:
In zuul runs the content of that file is already loaded.

Reply:
removed